### ISSUE TITLE
Objekte mit Schloss beim Projekt Import

### ISSUE THREAD

**Issue by vonCartman** [ioBroker/ioBroker.knx#259]
Hallo zusammen,

beim Projektimportieren werden mansche Objekte mit Schloss versehen.

Ich könnte keine Ursache finden, da scheint es mir "nach Zufallsprinzip" zu passieren.

ETS: 5.7.7
KNXAdapter: 2.0.13

![image](https://user-images.githubusercontent.com/65015881/191474614-f46c0f47-15f8-4331-b864-ec4094904e0f.png)

**Comment by Apollon77** [comment]
Ja, das sind "read only" States, die man laut den Onbjekt-Metadstane nicht beschreiben kann. Admin6 zeigt das jetzt über so ein Icon an damit man das besser sieht.

**Comment by vonCartman** [comment]
ja, die Frage ist warum?

Die Adressgruppe die in oberen Bild dargestellt unterschiedet sich nicht von anderen, und trotzdem bekomm sie Schloss, andere nicht.

**Comment by Apollon77** [comment]
Ok, DAS ist die frage für @chefkoch009 .. ich denke infos zu den settings dieses gerätes sind hilfreich. In jedem fall ist "write:false" im Objekt gesetzt

**Comment by chefkoch009** [comment]
Vermutlich hat das Objekt "read=true" und "write=true". Bitte mal in der ETS Prüfen.
